src/OidcClientSettings.ts

```diff
@@ -104,14 +104,15 @@ export interface OidcClientSettings {
     extraTokenParams?: Record<string, unknown>;
 
     /**
-     * Credentials used by fetch with the refresh request. (default: "same-origin")
+     * Will check the content type header of the response of the revocation endpoint to match these passed values (default: [])
      */
-    refreshTokenCredentials?: "same-origin" | "include" | "omit";
+    revokeTokenAdditionalContentTypes?: string[];
```

Comment:
1. please do not change the sequence, it makes it hard to see the diff
2. the rename refreshTokenCredentials to fetchRequestCredentials is a breaking change, maybe its better to leave this as is or we need to keep it and add the new one and merge them with this file. We can then also deprecate the old one...

Reply:
Okay, so I keep the refreshTokenCredentials even though i removed all usages of it in the code base? I get the idea I just want to make sure that i understand it correctly.

Reply:
yes, in src/OidcClientSettings.ts we then take the new `fetchRequestCredentials` if defined, else we fall-back to the deprecated `refreshTokenCredentials`

something like this
```
this.fetchRequestCredentials = fetchRequestCredentials ? fetchRequestCredentials
    : refreshTokenCredentials ? refreshTokenCredentials : "same-origin";
```

Nice to have: if both are defined print a warning or throw an Error. Not sure if worth...

Reply:
Ah yes of cause to still support the refreshToken. I pushed the changes. I think it makes sense with a warn if both properties are set. An error is a bit aggressive as the functionality will still work.